README.md

```diff
@@ -97,31 +97,39 @@ spec:
     - type: "ImageChange"
   source:
     dockerfile: |
-      FROM <driver-toolkit image> as builder
+      ARG DTK
+      FROM ${DTK} as builder
+
+      ARG KVER
 
       WORKDIR /build/
 
       RUN git clone https://github.com/openshift-psap/simple-kmod.git
 
       WORKDIR /build/simple-kmod
 
-      RUN make all install # [KVER=<kenel version if not default>] [KMODVER=<kernel module version if not default>]
+      RUN make all install KVER=${KVER}
 
       FROM registry.redhat.io/ubi8/ubi-minimal
 
-      # better to copy it from builder rather than pulling it from the internet so it work also with disconnected envs
-      COPY --from=builder /usr/bin/kmod /usr/bin/
-      RUN for link in /usr/bin/modprobe /usr/bin/rmmod; do \
-          ln -s /usr/bin/kmod "$link"; done
+      ARG KVER
+
+      # Required for installing `modprobe`
+      RUN microdnf install kmod
```

Comment:
Doesn't this require entitlements? lgtm otherwise.

Reply:
I don't think it does:
```
root kernel-module-management-midstream (e2e-cleanup) $ podman run -it registry.redhat.io/ubi8/ubi-minimal /bin/bash
[root@70756627fd6b /]# microdnf install kmod

(microdnf:11): librhsm-WARNING **: 08:58:27.834: Found 0 entitlement certificates

(microdnf:11): librhsm-WARNING **: 08:58:27.835: Found 0 entitlement certificates
Downloading metadata...
Downloading metadata...
Downloading metadata...
Package                                                                                            Repository                            Size
Installing:
 kmod-25-19.el8.x86_64                                                                             ubi-8-baseos                      129.2 kB
Transaction Summary:
 Installing:        1 packages
 Reinstalling:      0 packages
 Upgrading:         0 packages
 Obsoleting:        0 packages
 Removing:          0 packages
 Downgrading:       0 packages
Downloading packages...
Running transaction test...
Installing: kmod;25-19.el8;x86_64;ubi-8-baseos
Complete.

```

Reply:
OK then.